modules/scripts/startup-script/variables.tf

```diff
@@ -29,13 +29,25 @@ variable "region" {
   type        = string
 }
 
-
 variable "gcs_bucket_path" {
   description = "The GCS path for storage bucket and the object."
   type        = string
   default     = null
 }
 
+variable "bucket_viewers" {
+  description = "Service accounts or other IAM members (groups, users, domains) to which to grant read-only access to startup-script bucket"
+  type        = list(string)
+  default     = []
```

Comment:
What happens when you specify an empty array?

Reply:
No viewers are added. I've modified the variable description to read "Additional service accounts or groups, users, and domains to which to grant read-only access to startup-script bucket (leave unset if using default Compute Engine service account)"